docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of role Access Requests. This
+feature enables teams to enforce a zero standing privilege policy, while
+still allowing users to receive temporary access without manual approval.
+
+Support for automatic reviews of resource Access Requests is currently in
+development.
+
+## How it works
+
+Automatic reviews are triggered by Access Monitoring Rules. These rules instruct
+Teleport to monitor specific Access Requests and automatically submit a review
+when certain conditions are met. The conditions can be configured based on the
+requested roles, and based on the requesting user's Teleport traits.
+
+For example, an Access Monitoring Rule can configure automatic Access Request
+approval when a user with the Teleport traits `team: demo` requests access to
+the `access` role.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- This feature requires Teleport Identity Governance.
+
+## Step 1/4. Create a requester role and user
+
+In this example, we'll first create:
+- A role named `demo-access-request`, which allows requesting access to the
+`access` role.
+- A user named `demo-access-requester`, assigned the above role.
+
+### Create the role
+
+Create a role configuration file named `demo-role.yaml`:
+```yaml
+# demo-role.yaml
+kind: role
+version: v7
+metadata:
+  name: demo-access-request
+spec:
+  allow:
+    request:
+      roles:
+      - access
+      search_as_roles:
+      - access
+```
+
+Create the role with:
+```sh
+$ tctl create demo-role.yaml
+```
+
+### Create the user
```

Comment:
I'm wondering, should we split this section in 2 tabs that explains how to do this for a Teleport local vs SSO user?

- For Teleport local user, we tell them to create it and assign traits like you already have here.
- For SSO users, we tell them to ensure that IdP sends the required attribute (which will depend on IdP).

The reason is I think most people will want to use this feature with SSO users.

Reply:
I think we should keep this guide simple and straight forward with minimal configuration options. Following the [styles guide](https://www.notion.so/goteleport/Style-guide-160fdd3830be80a79e70f08df2df2a0a?pvs=4#160fdd3830be8098bd44cfc5361a9c57), I think it'd be better to have a separate access_monitoring_rules reference page that provides examples for all these different types of configurations, including SSO users.

Reply:
Sounds good. I think I would still at least mention here that if you're using SSO, then you can match on their attributes passed from an IdP. It's a useful context and most people reading this will be wondering.